tests/KubernetesClient.Tests/KubernetesClient.Tests.csproj

```
@@ -1,26 +1,26 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
-  <PropertyGroup>
-    <IsPackable>false</IsPackable>
-    <RootNamespace>k8s.Tests</RootNamespace>
-    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
-  </PropertyGroup>
-
-  <ItemGroup>
-      <PackageReference Include="MartinCostello.Logging.XUnit" Version="0.3.0" />
-      <PackageReference Include="FluentAssertions" Version="6.7.0" />
-      <PackageReference Include="Microsoft.Extensions.Logging" Version="6.0.0" />
-      <PackageReference Include="System.IO.Abstractions.TestingHelpers" Version="17.2.3" />
-      <PackageReference Include="System.Reactive" Version="5.0.0" />
-      <PackageReference Include="Nito.AsyncEx" Version="5.1.2" />
-      <PackageReference Include="Portable.BouncyCastle" Version="1.9.0" />
-  </ItemGroup>
-
-  <ItemGroup>
-
-    <PackageReference Include="coverlet.msbuild" Version="3.1.2">
-      <PrivateAssets>all</PrivateAssets>
-      <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
-    </PackageReference>
+    <PropertyGroup>
+        <IsPackable>false</IsPackable>
+        <RootNamespace>k8s.Tests</RootNamespace>
+        <TargetFrameworks>net5.0;net6.0</TargetFrameworks>
```

```suggestion
        <TargetFrameworks>netstandard2.0;netstandard2.1;net5.0;net6.0</TargetFrameworks>
```

what about this?
any idea to fix asp.net?

---

We can't target netstandard2.0/2.1 directly inside test project because , as stated in the [docs](https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-1-0)

> .NET Standard is a formal specification of .NET APIs that are available on multiple .NET implementations.

and depends by the framework implementation of the apis [docs](https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-1-0#specification).

xunit can't directly test netstandard because like other programs, require the netstandard api implementation to run. 
The additional property is the only trick i know to test netstandard implementation with xunit.

I tested on my machine the suggestion and, as i wrote, i can't build test project.

---

yup that is where i stopped.
prefer it than changing target during build.
it is not testing what is really shipped to nuget.